sourcecode/npm/components/src/components/AuthWrapper.jsx

```diff
@@ -58,12 +69,16 @@ const AuthWrapper = ({ children }) => {
                         onChange={(e) => setFirstName(e.target.value)}
                         margin="normal"
                         style={{marginRight: 10}}
+                        error={!!firstNameErrorText}
```

Comment:
Prefer using `firstNameErrorText !== ''`, `!!` saves a few characters but makes things harder to read.